plugins/delpoyment_stats/app/models/deployment_stats.rb

```
@@ -0,0 +1,63 @@
+require 'sqlite3'
```

rubocop (1.68.0)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)